nlp_apps.ipynb

```diff
@@ -522,7 +520,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 39,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#removing stopwords\n",
+    "from nltk.corpus import stopwords\n",
+    "stop_words = set(stopwords.words('english')) \n",
+    "\n",
+    "wordseqs = [w for w in wordseqs if not w in stop_words] "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#stemming and lemmatization\n",
+    "from nltk.stem.wordnet import WordNetLemmatizer\n",
+    "lmtzr = WordNetLemmatizer()\n",
```

Comment:
Again, we shouldn't use lemmatizers from third parties. Instead, we could have a lemmatizer within the repository, however basic it may be. The point of this repository is to be able to explain the underlying concepts of these algorithms, not directly import from other modules.

Reply:
Okay. I'll try to do that. Thank You!